packages/pool-metadata-gql/codegen.ts

```diff
@@ -0,0 +1,43 @@
+import { CodegenConfig } from "@graphql-codegen/cli";
+
+const BASE_URL = "https://api.thegraph.com/subgraphs/name/bleu-llc";
+
+export enum Network {
+  // mainnet = "mainnet",
+  // polygon = "polygon",
+  // arbitrum = "arbitrum",
+  goerli = "goerli",
+}
+
+export const ENDPOINTS = {
+  // [Network.mainnet]: `${BASE_URL}/balancer-v2`,
+  // [Network.polygon]: `${BASE_URL}/balancer-polygon-v2`,
+  // [Network.arbitrum]: `${BASE_URL}/balancer-arbitrum-v2`,
+  [Network.goerli]: `${BASE_URL}/balancer-pools-metadata`,
+};
```

Comment:
can you add a `#TODO(BAL-105): deploy to other networks` here?

Reply:
just to make sure everyone is on the same page - we'll only deploy to all networks once we finish the contracts (with support to DAO delegated pools) and ask for a review from Grants folks.